libkpod/pause.go

```diff
@@ -0,0 +1,44 @@
+package libkpod
+
+import (
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/pkg/errors"
+)
+
+// ContainerPause pauses a running container.
+func (c *ContainerServer) ContainerPause(container string) (string, error) {
+	ctr, err := c.LookupContainer(container)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to find container %s", container)
+	}
+
+	cStatus := c.runtime.ContainerStatus(ctr)
+	if cStatus.Status != oci.ContainerStatePaused {
+		if err := c.runtime.PauseContainer(ctr); err != nil {
+			return "", errors.Wrapf(err, "failed to pause container %s", ctr.ID())
+		}
+	}
+
+	c.ContainerStateToDisk(ctr)
```

Comment:
This can go inside the if statement above, we don't need to save state to disk if we did nothing

Reply:
fixed/moved.